tests/link/test_jax.py

```diff
@@ -978,6 +978,15 @@ def test_random():
     compare_jax_and_py(fgraph, [])
 
 
+def test_random_misc():
+    rng = shared(np.random.RandomState(123))
+    out = halfnormal(rng=rng)
+
+    fgraph = FunctionGraph([out.owner.inputs[0]], [out], clone=False)
+    with pytest.raises(NotImplementedError):
+        compare_jax_and_py(fgraph, [])
```

Comment:
We should create a custom no-op `RandomVariable` class for this test.  That way, there's no chance that JAX will implement `halfnormal` and cause this to fail.